logging/log.go

```diff
@@ -0,0 +1,157 @@
+// Copyright © 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package logging
```

Comment:
Can we call the package "log" instead of "logging"?

Reply:
I actually thought of that but there is a inbuilt log package in go which we are using. Need to replace that log package everywhere in mixer.

Reply:
yeah.. I figured.. That is why I had the other comment..